### DISCUSSION TITLE
Open Collective

### DISCUSSION THREAD

**Discussion by freearhey** [iptv-org/community#23] [Announcements]
To all @iptv-org/core: 

Any of you can get your share of the funds from our Open Collective [account](https://opencollective.com/iptv-org#category-BUDGET). Right now, each member can request up to 5,88% (100% / 17 members) of the organization's total budget. To do this, you just need to submit an invoice for the desired amount through this [form](https://opencollective.com/iptv-org/expenses/new). This process is described in more detail [here](https://docs.opencollective.com/help/expenses-and-getting-paid/submitting-expenses). Without such a request, unfortunately, it is not possible to transfer money to you. 

If you have any questions on this subject, leave them below and I will try to find an answer to them.

iptv-org/ledger: https://docs.google.com/spreadsheets/d/15Zobo3oj2dBLOBG3cYS94FJDt41hexpCRUFG1MmcifE/edit

**Comment by MapGuy11** [comment]
Will the funds expire and should be pulled immediately every month or can we let them just compound?

**Comment by freearhey** [comment]
There is no deadline, you can withdraw funds whenever you want.

**Comment by MapGuy11** [comment]
> There is no deadline, you can withdraw funds whenever you want.

Thanks 👍

**Comment by Daniel15** [comment]
What are some examples of reasons the money can be used for? What's the actual intention of the Open Collective?

**Comment by freearhey** [comment]
The money will be transferred to your bank account, so you can spend it however you want. Open Collective was chosen only because it allows to make the process of fundraising transparent for everyone.

**Comment by BellezaEmporium** [comment]
I give my part to @dct-infra for the infrastructure costs, he'll need it more than I do :)

**Comment by freearhey** [comment]
@LaneSh4d0w Okay, I'll keep that in mind.

**Comment by what-a** [comment]
...and I'd like my portion to go to @MapGuy11 to help with his server costs.

**Comment by MapGuy11** [comment]
> ...and I'd like my portion to go to @MapGuy11 to help with his server costs.

Are you sure? To be completely transparent I currently don't have any server costs due to Oracle free VM.

**Comment by what-a** [comment]
Okay. We'll it's there if you need it.

**Comment by MapGuy11** [comment]
> Okay. We'll it's there if you need it.

I thank you for your generosity it really helps! I will keep it in consideration.

**Comment by dct-infra** [comment]
$money? Where's my pen? I'm signing.
More seriously, thanks! @LaneSh4d0w too, this will help with setting up new exit points.

**Comment by sguinetti** [comment]
Thank you @freearhey. But the portion also goes to @MapGuy11 and @dct-infra. This option I consider appropriate for me.

**Comment by themak1985** [comment]
Hey folks - sorry I have been MIA - life has been challenging lately - I don't need any funds. it can go to you @freearhey

**Comment by freearhey** [comment]
@sguinetti Okay.

@themak1985 Thank you!

**Comment by freearhey** [comment]
@Central302 @RevGear Please add your email address to GitHub profile or create an account on [Open Collective](https://opencollective.com/) and write your username here so I can send you the invites.

@sguinetti Please check your email, there should already be an invite.

**Comment by Central302** [comment]
@freearhey 
Created Account. "CentralOSTL20"

**Comment by freearhey** [comment]
I created a kind of ledger so that I wouldn't get confused myself and so that anyone could quickly find out their current balance:

https://docs.google.com/spreadsheets/d/15Zobo3oj2dBLOBG3cYS94FJDt41hexpCRUFG1MmcifE/edit

**Comment by sguinetti** [comment]
@freearhey Thank you for provide information about this.

**Comment by freearhey** [comment]
@UltraHDR Please add your email address to GitHub profile or create an account on [Open Collective](https://opencollective.com/) and write your username here so I can send you the invite.

**Comment by freearhey** [comment]
@Alstruit Please add your email address to GitHub profile or create an account on [Open Collective](https://opencollective.com/) and write your username here so I can send you the invite.

**Comment by Alstruit** [comment]
> @Alstruit Please add your email address to GitHub profile or create an account on Open Collective and write your username here so I can send you the invite.

@freearhey I've created an account. The username should be the same.

**Comment by freearhey** [comment]
@Alstruit sent.

**Comment by freearhey** [comment]
@PopeyeTheSai10r Please add your email address to GitHub profile or create an account on [Open Collective](https://opencollective.com/) and write your username here so I can send you the invite.

**Comment by PopeyeTheSai10r** [comment]
I'll go ahead and create the account but my portion can go to you @freearhey.

**Comment by freearhey** [comment]
@PopeyeTheSai10r Thank you!

**Comment by freearhey** [comment]
The discussion has been moved to _iptv-org/iptv-org.github.io_ repository due to GitHub's decision to disable "Team Discussions": https://github.blog/changelog/2023-02-08-sunset-notice-team-discussions/

**Comment by freearhey** [comment]
@Carlinhos027 Please add your email address to GitHub profile or create an account on [Open Collective](https://opencollective.com/) and write your username here so I can send you the invite.

**Reply by Carlinhos027** [reply]
I just created an account. Can I send it to you via Discord?

**Reply by freearhey** [reply]
I don't use Discord.

**Reply by Carlinhos027** [reply]
Karlinhozz27 is my OC account

**Reply by freearhey** [reply]
Sent

**Comment by freearhey** [comment]
@CasperMcFadden95 Please add your email address to GitHub profile or create an account on [Open Collective](https://opencollective.com/) and write your username here so I can send you the invite.

**Comment by freearhey** [comment]
@amazeyourself @Archrootsda @brokenstream @sunyuanzhe Please add your email address to GitHub profile or create an account on [Open Collective](https://opencollective.com/) and write your username here so I can send you the invite.

**Comment by Archrootsda** [comment]
My username is the same as my GitHub username. Given the length of time I've been collaborating on the iptv-org project, I'm not the most suitable to receive this proportion. I redirect my part to @freearhey

**Reply by freearhey** [reply]
Thanks, but I assure you that you, like the rest of the team, have already contributed enough to the project to receive a share of the funds raised.

**Reply by Archrootsda** [reply]
Thanks @freearhey, but this time I'm forgoing the funds and I'm giving you the amount instead.

**Reply by freearhey** [reply]
Ok.

**Comment by brokenstream** [comment]
I feel the same as @Archrootsda. I have no meaningful use for the funds and really no interest in joining Open Collective. Appreciate the offer, but you or anyone else who needs it can have my share.

**Reply by freearhey** [reply]
Ok.

**Comment by freearhey** [comment]
@amazeyourself @CasperMcFadden95 @Pecaquito @StrangeDrVN @UltraHDR Please write your email address or [Open Collective](https://opencollective.com/iptv-org) username so I can send you an invite there.

**Reply by Pecaquito** [reply]
> @amazeyourself @CasperMcFadden95 @Pecaquito @StrangeDrVN @UltraHDR Please write your email address or [Open Collective](https://opencollective.com/iptv-org) username so I can send you an invite there.

Thank you very much for the offer, but my contribution is a way of helping with a project that is important to me, without expecting anything in return.

**Comment by freearhey** [comment]
@amazeyourself @CasperMcFadden95 @StrangeDrVN @UltraHDR Please write your email address or [Open Collective](https://opencollective.com/iptv-org) username so I can send you an invite there.

@tohenk Please check your email, there should already be an invite.

**Comment by amazeyourself** [comment]
my email is ***@***.***

**Reply by freearhey** [reply]
It looks like GitHub hides email addresses in comments. Maybe you already have an account on Open Collective? All I need is your username there to send you an invite.

**Comment by tohenk** [comment]
@freearhey , Can you re-send the invite to https://opencollective.com/tohenk?

**Reply by freearhey** [reply]
Sent.
